feature/featuretags/featuretags.go

```diff
@@ -90,9 +90,11 @@ var Features = map[FeatureTag]FeatureMeta{
 	"acme":          {"ACME", "ACME TLS certificate management", nil},
 	"appconnectors": {"AppConnectors", "App Connectors support", nil},
 	"aws":           {"AWS", "AWS integration", nil},
+	"bakedroots":    {"BakedRoots", "baked-in CA roots (LetsEncrypt) if system lacks them", nil},
```

Comment:
Optional nit: I think if this feature is enabled we bake in CA roots even if the system has them, right? It looks like the code always adds the baked-in roots if they're present.

If that's true, maybe we could leave out the condition (noticed mainly because it makes the generated comment read oddly).

```suggestion
	"bakedroots":    {"BakedRoots", "baked-in CA roots (LetsEncrypt)", nil},
```

Reply:
done